next.config.js

```diff
@@ -9,19 +9,23 @@ const nextConfig = {
     return [
       {
         source: '/api/:path*',
-        destination: `http://localhost:3000/api/:path*`,
+        destination: `http://${process.env.NEXT_PUBLIC_BACK_URL}/api/:path*`,
```

Comment:
The definition of the env variables should also include the `http://` prefix, because in the future the most probable thing is to change to `https://`. 

Please make this change for all the urls in this PR.